/**:
  ros__parameters: 
    wheel_diameter: 0.108
    cmd_max: 2000
    cmd_min: 1000
    steer_cmd_center: 1530
    steer_cmd_range: 900
    gear_ratio: 5.
    max_rpm: 16095.
    steer_angle_range: 0.8
    encoder_ticks: 8192
    map_filename: "map"